assets/javascripts/discourse/initializers/setup-subscriptions.js

```diff
@@ -23,6 +23,36 @@ export default {
           href: `/u/${user.username}/billing/subscriptions`,
           content: "Billing",
         });
+
+        if (user.admin) {
```

Comment:
I suggest returning early and not adding a large block of code for this if statement.

```
if (!user.admin) { return; }